src/document/json/counter.ts

```diff
@@ -90,6 +90,7 @@ export class Counter {
       );
     }
 
+    this.counter.increase(value);
```

Comment:
`Text` data type similarly self-updates its own CRDT element.
https://github.com/yorkie-team/yorkie-js-sdk/blob/eefad6391687ea757c00340eff692956d7be9b76/src/document/json/text.ts#L88-L93

Reply:
Yeap. This is intended behavior.
Refer to https://github.com/yorkie-team/yorkie-js-sdk/pull/441#issuecomment-1403182412